client/src/components/feeds/post/reactions/ReactionButton.css

```diff
@@ -104,4 +104,7 @@
     width: 1.2rem;
     height: 1.2rem;
   }
+  .PostFooter {
+    min-height: 1.2rem;
+  }
```

Comment:
It seems like this `min-height` wasn't enough to fix the layout shift on my screen, when I tried with `1.25rem` it seemed to be fixed. Could you try looking into if this value is ideal?

Reply:
I got the height from the height of the buttons (above). I tested in Safari, but it’s possible there’s an issue in whatever browser you’re using. one extra pixel isn’t a huge deal, so I’ll bump it.